CMakeLists.txt

```diff
@@ -4,6 +4,10 @@
 
 cmake_minimum_required(VERSION 3.5)
 PROJECT(GazeboYARPPlugins)
+option(USE_NEW_WRAPPERS "use new nws and nwc instead of old ones" OFF)
+if(USE_NEW_WRAPPERS)
+    add_definitions(-DUSE_NEW_WRAPPERS)
```

Comment:
I am afraid this could lead to subtle bugs, as it is a definition that it is used in headers, even if strictly speaking this headers are not public now. Perhaps it is better to specify this as `target_compile_definitions(<target> PUBLIC <define>)`, either for the plugin affected, or if you want it to be for all targets just add it in https://github.com/robotology/gazebo-yarp-plugins/blob/master/cmake/AddGazeboYarpPluginTarget.cmake . This avoid the risk that in the future a compilatin unit links a library whose ABI that expects `USE_NEW_WRAPPERS` to be defined, and does not define it, leading to runtime crashes. If something is not clear out of this jargon, feel free to ping me on teams.